[expo-sensors] Added isAvailable method to all sensors (#3257)
expo/expo: docs/versions/unversioned/sdk/accelerometer.md

Access the device accelerometer sensor(s) to respond to changes in
acceleration in 3d space.

### `Expo.Accelerometer.isAvailableAsync()`

Is the sensor enabled on the device.

#### Returns

- A Promise that resolves to a `boolean` denoting the availability of the sensor.

### `Expo.Accelerometer.addListener(listener)`

Subscribe for updates to the accelerometer.

#### Arguments

- **listener (_function_)** -- A callback that is invoked when an
  accelerometer update is available. When invoked, the listener is
  provided a single argument that is an object containing keys x, y,
  z.

#### Returns

> - An EventSubscription object that you can call remove() on when you

Reviewer: ```suggestion
- A subscription object that you can call `remove()` on when you
```

(the class name is an internal, implementation detail)
